packages/pharos-site/static/guidelines/pill.docs.tsx

```diff
@@ -0,0 +1,243 @@
+import PageSection from '@components/statics/PageSection.tsx';
+import BestPractices from '@components/statics/BestPractices.tsx';
+import { FC } from 'react';
+import Canvas from '../../src/components/Canvas';
+import { PharosPill, PharosLink } from '@ithaka/pharos/lib/react-components';
+
+const PillPage: FC = () => {
+  return (
+    <>
+      <PageSection
+        title="Pill"
+        description="A pill is used to display discrete information, create categorization, and label items. They can be interactive (dismissible) or informative."
+        isHeader
+        storyBookType="components"
+      >
+        <Canvas>
+          <PharosPill>Email address</PharosPill>
+          <PharosPill dismissible>Topic: Art History</PharosPill>
+        </Canvas>
+      </PageSection>
+      <PageSection title="Usage">
+        <p>
+          Pills are used to represent a discrete piece of information (i.e. email address, username,
+          role name) that the user can interact with, often in an input or a list format.
+        </p>
+        <PageSection subSectionLevel={1} title="When to use pills">
+          <ul>
+            <li>When you want to add/remove discrete items to a list</li>
+            <li>When you need to apply dynamic filters</li>
+            <li>When you want to display tags or labels on an item</li>
+          </ul>
+        </PageSection>
+        <PageSection subSectionLevel={1} title="When not to use pills">
+          <ul>
+            <li>
+              When needing multiple interactions within a single pill, consider a card or a tile
+            </li>
+            <li>When needing to display long explanatory text. Consider using a caption instead</li>
+          </ul>
+        </PageSection>
+      </PageSection>
+      <PageSection title="Best practices">
+        <BestPractices
+          Do={
+            <ul>
+              <li>Stack pills if screen width is constrained</li>
+              <li>Limit pill text to 1-2 words long and reflect a discrete name for the data</li>
+            </ul>
+          }
+          Dont={
+            <ul>
+              <li>Do not wrap contents inside the pill</li>
+              <li>Do not use a pill to replace a button</li>
+              <li>Do not use a pill to navigate to another page</li>
+            </ul>
+          }
+        />
+      </PageSection>
+      <PageSection title="Content guidelines">
+        <PageSection subSectionLevel={1} title="Pill groups">
+          <ul>
+            <li>
+              Avoid mixing and matching colors unless the different color of the pill conveys
+              important meaning about the data inside
+            </li>
+            <li>
+              If including an icon on the left-hand side, make certain the icon is consistent for
+              all data in the same category (i.e. if including a mail icon next to an email address,
```

Comment:
This is a bit wordy for a parenthetical, consider using "As an example, ..." as a full next sentence instead.